Automatic/move-images-to-other-tab.ini

```diff
@@ -1,7 +1,16 @@
 [Command]
 Name=Move Images to Other Tab
-Input=image/png
 Automatic=true
-Remove=true
 Icon=\xf03e
 Tab=&Images
```

Comment:
Oh, this is why **also text gets move**. "Tab" should be avoided, it gets handled by the command.